### ISSUE TITLE
Update the api endpoint for login and auth

### ISSUE THREAD

**Issue by johngitahi** [Shirakumo/plaster#5]
The current plaster.el endpoint being used is api/simple-auth but the plaster service uses api/auth for authentication. Without changing that emacs shows an error when trying to log in.

**Comment by Shinmera** [comment]
Yeah it should be updated to use oAuth rather than cookies, as well.
